src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -182,19 +194,14 @@ public void doGet_returnsCommentsForSpecificLecture() throws ServletException, I
   }
 
   private Entity createTestCommentEntity(int lectureId) {
-    Entity commentEntity = new Entity(CommentUtil.KIND);
-    Key lecture = KeyFactory.createKey(/* kind= */ "Lecture", lectureId);
-    commentEntity.setProperty(CommentUtil.LECTURE, lecture);
-    commentEntity.setProperty(CommentUtil.PARENT, null);
-    commentEntity.setProperty(CommentUtil.TIMESTAMP_MS, 0);
-    // Most properties here are not tested, but are required by the AutoValue class so must be
-    // specified.
-    commentEntity.setProperty(
-        CommentUtil.AUTHOR, new User("untestedAuthor@example.com", "untested.com"));
-    commentEntity.setProperty(CommentUtil.CONTENT, "Untested content");
-    commentEntity.setProperty(CommentUtil.CREATED, new Date(Clock.systemUTC().millis()));
-
-    return commentEntity;
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
+    long timestampMs = 123;
```

Comment:
can you make `timestampMs` a global variable since it is used in different tests?

Reply:
Done